### ISSUE TITLE
Getting Authelia to play nice with Traefik2, VLC and protected media endpoint

### ISSUE THREAD

**Issue by samip5** [authelia/authelia#1770]
I found out that you can use basic auth directly with authelia, but I was unable to find a way to get it to work properly?

It will only return a 401 response, but nothing else and VLC will not prompt for credentials nor accept them when encoded into the URL like `https://<username>:<password>@<domain>`.

curl with username and password encoded in url:
```
* Using HTTP2, server supports multi-use
* Connection state changed (HTTP/2 confirmed)
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* Server auth using Basic with user '<censored>'
* Using Stream ID: 1 (easy handle 0x176883a694b0)
> GET / HTTP/2
> Host: <censored-domain>
> Authorization: Basic <censored>
> User-Agent: curl/7.64.0
> Accept: */*
>
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
* Connection state changed (MAX_CONCURRENT_STREAMS == 250)!
< HTTP/2 401
< content-type: text/plain; charset=utf-8
< date: Sun, 28 Feb 2021 16:24:30 GMT
< referrer-policy: same-origin
< x-content-type-options: nosniff
< x-xss-protection: 1; mode=block
< content-length: 12
<
* Connection #0 to host <censored> left intact
Unauthorized
```

My forwardAuth is set as:
```yml
[http.middlewares]
  [http.middlewares.basic-authelia.forwardAuth]
    address = "http://authelia:9091/api/verify?auth=basic"
    trustForwardHeader = true
    authResponseHeaders = ["Remote-User", "Remote-Groups"]
```

Basically going off this example for Heimdall and Traefik 2.x: https://www.authelia.com/docs/deployment/supported-proxies/traefik2.x.html

**Comment by samip5** [comment]
Looking at the code in https://github.com/authelia/authelia/blob/master/internal/handlers/handler_verify.go#L236, it seems that it should return with the WW-Authenicate header which it does not?

**Comment by james-d-elliott** [comment]
Which version of Authelia are you using?

**Comment by samip5** [comment]
> Which version of Authelia are you using?

Latest, available as a docker tag authelia/authelia:latest.

**Comment by james-d-elliott** [comment]
You'd have to be using `master` or wait until 4.27.0 comes out.

**Comment by samip5** [comment]
> You'd have to be using `master` or wait until 4.27.0 comes out.

Is new version released every two weeks or so?

**Comment by james-d-elliott** [comment]
> 
> 
> > You'd have to be using `master` or wait until 4.27.0 comes out.
> 
> Is new version released every two weeks or so?

Generally it ends up around that timeframe, sometimes a week, sometimes two or more. We generally release based on our backlog. So if we have a few features to release we try to release minor versions after merging them all. Once we get a minor release out bugfixes happen regularly. Based on the status of our work I'd guess in the next few days we[ll release, just waiting on some code reviews.

**Comment by james-d-elliott** [comment]
Fixed by 4.27.0 I believe. If not please let me know by replying and reopening this. Or feel free to ask questions in the same way, or via matrix/discord.
